protocols/bgp/server/withdraw_test.go

```diff
@@ -111,7 +111,9 @@ func TestWithDrawPrefixesAddPath(t *testing.T) {
 	}
 	for _, tc := range testcases {
 		buf := bytes.NewBuffer([]byte{})
-		opt := &packet.Options{}
+		opt := &types.Options{
+			AddPathRX: true,
+		}
 		err := withDrawPrefixesAddPath(buf, opt, tc.Prefix, tc.Path)
```

Comment:
we should consider renaming it to withdrawPrefixes and let this method decide which type of withdraw message is needed internaly based on the options passed

Reply:
nothing to do here for now